docs/markdown/Adding-new-projects-to-wrapdb.md

```diff
@@ -1,8 +1,8 @@
-# Adding new projects to wrap
+# Adding new projects to WrapDB
 
-**If you don't have permissions to do something on this page, please
-  open issue against https://github.com/mesonbuild/wrapweb/issues to
-  let us know that you want to start new project.**
+If you don't have permissions to do something on this page, please
```

Comment:
The "don't have permissions to do something on this page" is actually leftover from the time when this doc was hosted in the Github wiki. The whole bit should be removed as it is confusing and replace with something simpler like "If you would like to submit a new Wrap to the WrapDB, please file an issue on the wrapweb bug tracker".

Reply:
Thanks for the feedback. I will fix it.